paddle/fluid/framework/details/computation_op_handle.cc

```diff
@@ -20,18 +20,30 @@ namespace paddle {
 namespace framework {
 namespace details {
 ComputationOpHandle::ComputationOpHandle(ir::Node *node, Scope *scope,
-                                         platform::Place place)
+                                         platform::Place place,
+                                         size_t scope_idx)
     : OpHandleBase(node),
       op_(framework::OpRegistry::CreateOp(*node->Op())),
       scope_(scope),
-      place_(place) {}
+      place_(place),
+      scope_idx_(scope_idx) {}
```

Comment:
It seems that `scope_idx_ ` is not used in `ComputationOpHandle `.

Reply:
For future use. Have deleted.